ui/dashboards/src/views/ViewDashboard/index.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export * from './ViewDashboard';
```

Comment:
Do you think it's worth also exporting DashboardApp here? In case we, or someone in the future, wants to hook in a level lower than ViewDashboard

Reply:
Yeah, good point. I was gonna wait til we crossed that bridge, but I could definitely see it. I think in that case we'd move it to `components` and do it?